lib/va_profile/veteran_status/service.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+require 'common/client/concerns/monitoring'
+require 'common/client/errors'
+require 'va_profile/service'
+require 'va_profile/stats'
+require 'va_profile/models/veteran_status'
+require_relative 'configuration'
+require_relative 'veteran_status_response'
+
+module VAProfile
+  module VeteranStatus
+    class Service < VAProfile::Service
+      include Common::Client::Concerns::Monitoring
+
+      configuration VAProfile::VeteranStatus::Configuration
+
+      OID = '2.16.840.1.113883.3.42.10001.100001.12' # double check swagger
+      AAID = '^NI^200DOD^USDOD' # double check swagger.
+
+      # GET's a user's veteran status info from the VAProfile API
+      # If a user is not found in VAProfile, an empty VeteranStatusResponse with a 404 status will be returned
+      # @return [VAProfile::VeteranStatus::VeteranStatusResponse] response wrapper around a veteran_status object
+      def get_veteran_status
+        with_monitoring do
+          edipi_present!
+
+          response = perform(:post, identity_path, VAProfile::Models::VeteranStatus.in_json)
+          VeteranStatusResponse.from(@current_user, response)
```

Comment:
Why pass `@current_user` here if it's ignored on the `VeteranStatusResponse` class?